routers/api/v1/repo/branch.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2014 The Gogs Authors. All rights reserved.
```

Comment:
It's `2016`.

Also add this copyright content to `models/repo_branch.go` file.